src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -33,6 +33,16 @@
 
 namespace Microsoft.IdentityModel.Tokens
 {
+    /// <summary>
+    /// Definition for AcceptedAlgorithmValidator
```

Comment:
Can this just be called AlgorithmValidator? That seems to better fit the naming model of the other delegates.